src/components/Synthetics/TableOptionsFilter/TableOptionsFilter.tsx

```diff
@@ -46,8 +55,10 @@ export function TableOptionsFilter<T>({
   label,
   ItemComponent,
   popupPlacement,
+  beforeContent,
+  extraIsActive,
```

Comment:
I think you were looking for `forceIsActive` for the name. 

Can be also `overrideIsActive` but should behave a bit differently: `overrideIsActive ?? (multiple ? Boolean(value?.length) : Boolean(value))`

Reply:
done